src/Platform/Microsoft.Testing.Platform/IPC/Serializers/BaseSerializer.cs

```diff
@@ -68,6 +92,22 @@ protected static void WriteLong(Stream stream, long value)
         stream.Write(bytes);
     }
 
+    protected static void WriteShort(Stream stream, short value)
+    {
+        Span<byte> bytes = stackalloc byte[sizeof(short)];
```

Comment:
for coherency or we use numbers everywhere of `sizeof(...)` sizeof is expanded at compile time https://sharplab.io/#v2:EYLgHgbALANALiAlgGwD4AEBMBGAsAKHQGYACLEgYRIG8CT6zTEA7OEgWQAoBKGuhgegDsJAM6IAXgFMA9gDNOLONwDc/egF8CGoA===

pls substitute all places in this file where we have the number in favour of `sizeof(...)` that looks to me more explicit